app-testing/tests/protocol_analyze_test.py

```diff
@@ -40,6 +40,7 @@ def test_analyses(
     # Upload labware if any
     if protocol.custom_labware:
         for labware in protocol.labware_paths:
+            print(labware)
```

Comment:
So there are cleaner ways to get Rich (the package) output but this is how it is set up right now.
```suggestion
            console.print(labware)
```